ravedude/Cargo.toml

```
@@ -21,3 +21,4 @@ serde = { version = "1.0.197", features = ["serde_derive"] }
 toml = "0.8.11"
 either = "1.10.0"
 clap = { version = "4.0.0", features = ["derive", "env"] }
+terminal-link = "0.1.0"
```

Personally, I am not a fan of doing such fancy things here, especially as it is only for a very niche case (when `avrdude` is not installed).  Worst case, someone has a terminal that doesn't support such links and they get broken output...

But if you insist, I am fine with keeping this in for Windows.  In that case, please make the dependency windows-only, though:

```suggestion

[target.'cfg(target_os = "windows")'.dependencies]
terminal-link = "0.1.0"
```

---

From what I've found, most modern (5 or so years) terminal will work.

And I tested on original windows command prompt (and not windows terminal), it seems to just hide the link part and the text is shown as normal without any hyperlink reference.